specification/cognitiveservices/data-plane/AnomalyDetector/preview/v2.0-preview/RootCauseAnalysis.json

```diff
@@ -0,0 +1,2143 @@
+            }
+          },
+          "400": {
+            "description": "Bad Request",
+            "schema": {
+              "$ref": "#/definitions/MultiDimRCAListDataSetErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "description": "String identical to error code.",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List root cause analysis data set": {
+            "$ref": "./examples/ListRootCauseAnalysisDataSet.json"
+          }
+        }
+      }
+    },
+    "/rootCauseAnalysis/dataSets/{dataSetId}": {
+      "get": {
+        "tags": [
+          "RootCauseAnalysisDataSet"
+        ],
+        "summary": "Get a root cause analysis data set by its id",
+        "operationId": "GetRootCauseAnalysisDataSet",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "dataSetId",
+            "description": "The root cause analysis data set unique id",
+            "required": true,
+            "type": "string",
+            "format": "uuid"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/RootCauseAnalysisDataSet"
+            }
+          },
+          "404": {
+            "description": "Not Found",
+            "schema": {
+              "$ref": "#/definitions/MultiDimRCAGetDataSetErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "description": "String identical to error code.",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get a root cause analysis data set by its id": {
+            "$ref": "./examples/GetRootCauseAnalysisDataSet.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "RootCauseAnalysisDataSet"
+        ],
+        "summary": "Delete a root cause analysis data set",
+        "operationId": "DeleteRootCauseAnalysisADataSet",
+        "parameters": [
+          {
+            "in": "path",
+            "name": "dataSetId",
+            "description": "The root cause analysis data set unique id",
+            "required": true,
+            "type": "string",
+            "format": "uuid"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "The resource is deleted successfully or does not exist."
+          }
+        },
+        "x-ms-examples": {
+          "Delete a root cause analysis data set": {
+            "$ref": "./examples/DeleteRootCauseAnalysisADataSet.json"
+          }
+        }
+      }
+    },
+    "/rootCauseAnalysis/dataSets/{dataSetId}:upload": {
+      "post": {
```

Comment:
What is the options, if user failed to receive the response of the POST (due to timeout etc.)? Will retry work?

Reply:
We have validation in API that same partition Id cannot be uploaded more than once